crud/src/main/java/com/apirest/crud/controller/userController.java

```diff
@@ -21,24 +19,60 @@ public class userController {
     @Autowired
     private userImplService userservice;

```

Comment:
userImplService, for classes ple always follow standards, this should start with Capital letter